lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +67,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || @options.on_failure
```

Comment:
options is just a hash, so we can't access attributes in this way. Try `@options[:on_failure]`